lib/ansible/modules/network/ios/_ios_template.py

```diff
@@ -147,9 +147,7 @@ def main():
     candidate = NetworkConfig(contents=module.params['src'], indent=1)
 
     result = {'changed': False}
-    warnings = list()
-    check_args(module, warnings)
-    result['warnings'] = warnings
+    check_args(module)
```

Comment:
You cannot imagine, but people do this all the time. I often grep for strings to get quick examples (i.e. warning in this case). And you don't know a module is deprecated unless you are specifically looking for it. So to me it is still important and relevant to make these changes to deprecated modules.